README.md

```diff
@@ -404,7 +398,6 @@ By default Jest ignores everything in `node_modules`. This setting prevents Jest
 ## Known Limitations
 ### Known limitations for TS compiler options
 - You can't use `"target": "ES6"` while using `node v4` in your test environment;
```

Comment:
I think we can put here that we don't support `node v4`.

Reply:
Agreed. Tho while updating the README I tried to keep my update as small as possible, and looked quickly to the stuff only related to adding the wiki troubleshooting link. Shouldn't we keep that for another PR?